README.md

```diff
@@ -0,0 +1,64 @@
+# MAGeTbrain Pipeline
+
+A Nextflow implementation of the [Multiple Automatically Generated Templates (MAGeT) brain](https://github.com/CobraLab/MAGeTbrain) segmentation pipeline.
+
+> Pipitone J, Park MT, Winterburn J, et al. Multi-atlas segmentation of the whole hippocampus
+> and subfields using multiple automatically generated templates. Neuroimage. 2014;
+
+> M Mallar Chakravarty, Patrick Steadman, Matthijs C van Eede, Rebecca D Calcott, Victoria Gu, Philip Shaw, Armin Raznahan, D Louis Collins, and Jason P Lerch.
+> Performing label-fusion-based segmentation using multiple automatically generated templates. Hum Brain Mapp, 34(10):2635–54, October 2013. (doi:10.1002/hbm.22092)
+
+## Prerequisites
+
+- [Nextflow](https://www.nextflow.io/) (version 20.07.1 or later)
+- [Docker](https://www.docker.com/) or [Singularity](https://sylabs.io/singularity/)
+- MINC Toolkit (provided in the Docker container) _optional_
+
+## Quick Start
+
+> [!IMPORTANT]  
+> All images should be in NIfTI format (`.nii.gz`)  
+> `.mnc` can be converted to NIfTI using `mnc2nii` from minc-toolkit-v2  
+> see below in section [Converting MINC to NIFTI](#converting-minc-to-nifti)  
+> [link to docs](https://bic-mni.github.io/man-pages/man/mnc2nii)
+
+1. Clone the repository:
+
+```bash
+git clone https://github.com/CoBrALab/MAGeTBrain-nextflow.git
+cd MAGeTBrain-nextflow
+```
+
+2. Input structure
+   Atlases and images should be structure in an input directory as follows:
+
+```bash
```

Comment:
The structure for label-names csv is not included here.

It should be something like names_label_<labelname>.csv, one per label file type in the atlas directory.